applications/zigbee_weather_station/README.rst

```diff
@@ -0,0 +1,519 @@
+
+The following Kconfig option values can be set to custom values:
+
+* :kconfig:`CONFIG_FIRST_WEATHER_CHECK_DELAY_SECONDS` - This option defines the delay from the moment of the application initialization to the first sensor data readout.
+  The value is provided in seconds, with the default value set to ``5``.
+
+* :kconfig:`WEATHER_CHECK_PERIOD_SECONDS` - This option defines the period of cyclic sensor readouts after the first readout.
+  The value is provided in seconds, with the default value set to ``60``.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/zigbee_weather_station`
+
+.. include:: /includes/build_and_run.txt
+
+Selecting a build type
+======================
+
+Before you start testing the application, you can select one of the :ref:`zigbee_weather_station_app_build_types`, depending on your building method.
+
+Selecting a build type in |VSC|
+-------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_vsc_start
+   :end-before: build_types_selection_vsc_end
+
+Selecting a build type in SES
+-----------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_ses_start
+   :end-before: build_types_selection_ses_end
+
+Selecting a build type from command line
+----------------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_cmd_start
+   :end-before: For example, you can replace the
+
+For example, you can replace the *selected_build_type* variable to build the ``release`` firmware for ``thingy53_nrf5340_cpuapp`` by running the following command in the project directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -b thingy53_nrf5340_cpuapp -d build_thingy53_nrf5340_cpuapp -- -DCONF_FILE=prj_release.conf
+
+The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory for the build files.
+
+.. note::
+   If the selected board does not support the selected build type, the build is interrupted.
+   For example, if the ``shell`` build type is not supported by the selected board, the following notification appears:
+
+   .. code-block:: console
+
+      File not found: ./ncs/nrf/applications/zigbee_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
+
+Testing
+=======
+
+Preliminary testing information
+    * The testing procedure assumes you are using the ``debug`` :ref:`build type <zigbee_weather_station_app_build_types>`, as it provides more feedback with **LED (LD1)** and logs through the USB console.
+      The steps related to the ``debug`` build type are marked as "Optional ``debug`` build type step".
+      These steps are not required when using the ``release`` build type.
+    * Wireshark with :ref:`ug_zigbee_tools_sniffer` is used for packet capturing.
+      This is an optional setup, but Wireshark response frame examples are provided in the Testing steps as reference.
```

Comment:
This can actually be removed, and step 1 of the testing procedure can be reworked -- see comment to line 229.
```suggestion
```

Reply:
Done